src/main/java/org/schemaspy/connection/SemicolonSeparated.java

```diff
@@ -0,0 +1,39 @@
+package org.schemaspy.connection;
+
+import java.util.Properties;
+import java.util.StringTokenizer;
+
+/**
+ * Specifies connection properties to use in the format:
+ * <code>key1\=value1;key2\=value2</code><br>
+ * user (from -u) and password (from -p) will be passed in the
+ * connection properties if specified.<p>
+ * This is an alternative form of passing connection properties than by file
+ */
+public class SemicolonSeparated {
+
+    private final String properties;
+    private static final String ESCAPED_EQUALS = "\\=";
+
+    /**
+     * @param properties string with key\\=value pairs separated by ; of connection properties
+     */
+    public SemicolonSeparated(String properties) {
+        this.properties = properties;
+    }
+
+    public Properties setConnectionProperties() {
```

Comment:
Why is this set? Shouldn't even be get, since it returns a new Properties.

Reply:
I agree but won't do everything at once. Below is whatit looked like previously.
https://github.com/schemaspy/schemaspy/blob/0f31217787f17b2cf710e1962f0df9a249eb2d77/src/main/java/org/schemaspy/ConnectionProperties.java#L33-L36